test/helpers/browser.js

```diff
@@ -1,23 +1,37 @@
-var jsdom = require('jsdom').jsdom
+var jsdom = require('jsdom').jsdom;
+var fs = require('fs');
 
-var exposedProperties = ['window', 'navigator', 'document']
+// jsdom.defaultDocumentFeatures = {
```

Comment:
This file could be cleaned (comments & console.log), unless you want to keep these things commented out for some reason.